.github/workflows/sync-fork.yml

```diff
@@ -0,0 +1,30 @@
+name: Sync to Upstream
+
+on:
+  schedule:
+    - cron:  '13 5 * * *'
+    # scheduled at 05:13 every day
+  workflow_dispatch:
+
+jobs:
+  sync_with_upstream:
+    runs-on: ubuntu-latest
+    name: Sync HEAD with upstream latest
+
+    steps:
+    # Step 1: run a standard checkout action, provided by github
+    - name: Checkout HEAD
+      uses: actions/checkout@v2
+      with:
+        ref: master
+
+    # Step 2: run this sync action - specify the upstream repo, upstream branch to sync with, and target sync branch
+    - name: Pull upstream changes
+      id: sync
+      uses: aormsby/Fork-Sync-With-Upstream-action@master
+      with:
+        upstream_repository: augustogoulart/awesome-pytest
+        upstream_branch: master
+        git_pull_args: --unshallow
+        target_branch: master                       # optional
+        github_token: ${{ secrets.GITHUB_TOKEN }}   # optional, for accessing repos that require authentication
```

Comment:
@cleder, what’s this intended for?

Reply:
This is to keep Forks automatically synchronized to upstream (your repo) https://dev.to/github/sync-forks-to-upstream-using-github-actions-gle

Reply:
It's an interesting idea, but I'm not sure if I like that.

I would rather delegate to fork-owners updating their repos, especially if they are building something out of upstream and don't intend to sync at all. 

Let's remove that for now and I'll merge the other changes. 

Thanks for the PR!